cmd/comment/handler.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"context"
+	"github.com/Tiktok-Lite/kotkit/cmd/comment/pack"
+	"github.com/Tiktok-Lite/kotkit/internal/db"
+	"github.com/Tiktok-Lite/kotkit/internal/model"
+
+	"github.com/Tiktok-Lite/kotkit/kitex_gen/comment"
+	"github.com/Tiktok-Lite/kotkit/pkg/helper/constant"
+	"github.com/Tiktok-Lite/kotkit/pkg/log"
+)
+
+// CommentServiceImpl implements the last service interface defined in the IDL.
+type CommentServiceImpl struct{}
+
+// CommentAction implements the CommentServiceImpl interface.
+func (s *CommentServiceImpl) CommentAction(ctx context.Context, req *comment.DouyinCommentActionRequest) (resp *comment.DouyinCommentActionResponse, err error) {
+	logger := log.Logger()
+
+	res := &comment.DouyinCommentActionResponse{
+		StatusCode: constant.StatusOKCode,
+		StatusMsg:  "success",
+		Comment:    nil,
+	}
+	claims, err := Jwt.ParseToken(req.Token)
+	if err != nil {
+		logger.Errorf("Error occurs when parsing token. %v", err)
+		res.StatusMsg = "token错误"
+		return res, err
```

Comment:
```suggestion
		return res, nil
```
在rpc服务端不要返回error，要不然会被kitex认定为rpc链路错误，而其实它应该是业务错误，所以应当在rpc客户端处理error

Reply:
另外这里应当新建一个res，如果不新建的话就引用了上面的res，返回结果是`success`，是不符合逻辑的